units/provisioning_setup_host.go

```diff
@@ -502,6 +506,7 @@ func (j *setupHostJob) provisionHost(ctx context.Context, settings *evergreen.Se
 				"host_id":   j.host.Id,
 				"job":       j.ID(),
 			}))
+			j.host.RevokeGithubTokens(ctx)
```

Comment:
Did we test that this revokation works with `evergreen fetch`? It should be invalid to revoke the token here because the `evergreen fetch` runs in the background on the host, meaning it can revoke while `evergreen fetch` is still running.

If we really want to revoke the tokens safely, we could add an option to `evergreen fetch` to auto-revoke the tokens when the command is finished. That way, there's no need for the app servers to try to figure out when the host has finished running the command.